Gemfile

```diff
@@ -51,6 +51,7 @@ gem 'okcomputer'
 gem 'pg'
 gem 'rails', '~> 6.0.3', '>= 6.0.3.2'
 gem 'redis', '~> 4.0'
+gem 'reform-rails', '~> 0.2.0'
```

Comment:
would love 5 words or less explaining what this gem is giving us

Reply:
Its a way of having form-models.  We use the reform pattern in argo (but we didn't use reform for obscure reasons): https://github.com/sul-dlss/argo/blob/d317403b9bfd8abbe37ab97947abf2dba03e67f0/app/forms/base_form.rb#L3